scss/chat/_chat-list-item.scss

```diff
@@ -327,7 +327,7 @@
   --size: 50px;
   --iconSize: 20px;
   position: fixed;
-  z-index: 20;
+  z-index: 1; // local z-index scoped to the chatlist
```

Comment:
You could also move these to the variables file. Name the key accordingly to the comment, e.g. `'local-scope-chatlist'` or similar. This way we don't need these comments and we really have all z-index values in one file.